datetime.go

```diff
@@ -34,6 +35,18 @@ var formatReplacements = []formatReplacement{
 	{input: "TZM", output: "00"},
 }
 
+func timeToString(t time.Time, dateTimeType string, params map[string]*string) (string, error) {
+	sfFormat, err := dateTimeInputFormatByType(dateTimeType, params)
+	if err != nil {
+		return "", err
```

Comment:
shouldn't we log these error?

Reply:
Why? We don't log everywhere. Root caller should log error propagated to the top. Otherwise we would have multiple logs with the same text.